README.md

```diff
@@ -14,10 +14,22 @@ In the past, these have been in one launch script (exporter.py) but this was not
 
 1. CLI
 
-    Either specify the vars via environment or cli params. Here an example start command for both, inventory and exporter:
+    Either specify the vars via environment or cli params. Because the inventory and the exporter is running seperately,
+    you need to enter the Docker container at least twice. Start the Docker container:
```

Comment:
remove `Docker` the second time, should be sufficient to refer to it as container